qualtran/bloqs/qubitization_walk_operator.py

```diff
@@ -92,6 +94,11 @@ def signature(self) -> Signature:
     def reflect(self) -> ReflectionUsingPrepare:
         return ReflectionUsingPrepare(self.prepare, control_val=self.control_val, global_phase=-1)
 
+    @cached_property
+    def sum_of_lcu_coefficients(self) -> Optional[float]:
+        r"""value of $\lambda$, i.e. sum of the coefficients $w_l$."""
```

Comment:
Let's clarify by writing "sum of absolute values of coefficients. " instead. In this case, the docstring of the class does clarify that each $w_l$ is a positive integer so it's just a sum of coefficients.

Reply:
fixed